### ISSUE TITLE
support -m

### ISSUE THREAD

**Issue by BjornMelgaard** [commitizen/cz-cli#454]
Expected behavior:
```
git cz -m "somemessage"
```
results in filled `Write a short, imperative tense description of the change:`

**Comment by pawamoy** [comment]
Yes, too disruptive right now. I'm used to always type `gc -am "my commit message"`. I would like to be able to do the same with `gz -am "my commit message"`, then choose the type and scope of the commit. The short message and long description would be prefilled with the given string.

**Comment by jedwards1211** [comment]
yeah I really only want to use commitizen as an aid for other people who are committing to my repo, I'm used to the message format so it's faster for me to type it out manually.

**Comment by jedwards1211** [comment]
Okay what I figured out for the husky hooks at least is

```
"prepare-commit-msg": "egrep -q '\\S' .git/COMMIT_EDITMSG || (exec < /dev/tty && git cz --hook || true)"
```

**Comment by jedwards1211** [comment]
okay finally figured out a work around for husky hooks:

```
"prepare-commit-msg": "grep -qE '^[^#]' .git/COMMIT_EDITMSG || (exec < /dev/tty && git cz --hook || true)",
```
I guess this wouldn't behave as expected if you do something silly like `-m " "` but good enough
